materialize-databricks/.snapshots/TestSpecification

```diff
@@ -138,7 +189,8 @@
         "type": "object",
         "title": "Advanced Options",
         "description": "Options for advanced users. You should not typically need to modify these.",
-        "advanced": true
+        "advanced": true,
+        "order": 6
```

Comment:
A few of these have `order` set in their configurations but others don't - just wondering if there is a specific reason for that?

Reply:
I haven't set order for `dbt_job_trigger` at all, some connectors' configurations generally have `order` while others don't... the diff is for other properties of those connectors